laravel/app/Models/Option.php

```diff
@@ -28,4 +28,24 @@ public function anesthesiologist()
     {
         return $this->belongsTo(Anesthesiologist::class);
     }
+
```

Comment:
missing option -> assignment relationship. I think it's a one to one.

Reply:
Completely missed that, going to rename the option column in assignment to option_id and get the relationship set up